### DISCUSSION TITLE
error github token is missing

### DISCUSSION THREAD

**Discussion by vijyantg** [suzuki-shunsuke/tfcmt#746] [Announcements]
https://suzuki-shunsuke.github.io/tfcmt/github-enterprise
I am using GHE and I have tried all the possible way to export Github_token 
but I haven't tried using above link : where to pass this ghe_base_url

**Comment by suzuki-shunsuke** [comment]
I can't verify using GitHub Enterprise because I haven't used it.
But the error `github token is missing` has nothing to do with GitHub Enterprise.

https://github.com/search?q=repo%3Asuzuki-shunsuke%2Ftfcmt%20%22github%20token%20is%20missing%22&type=code

https://github.com/suzuki-shunsuke/tfcmt/blob/b2910c68140297c334a4766722706ac288571762/pkg/notifier/github/client.go#L72-L83

Please check if the environment variable `GITHUB_TOKEN` is set.

And please tell us the following information.

- tfcmt.yaml
- command and output and exit code
- tfcmt version (the result of `tfcmt -v`)
- Environment (OS and CPU Architecture)

**Reply by vijyantg** [reply]
<img width="570" alt="image" src="https://user-images.githubusercontent.com/81227957/233380053-6ea068b1-193d-4bd6-a5e0-30f023e4593b.png">
<img width="685" alt="image" src="https://user-images.githubusercontent.com/81227957/233380271-e2eb6710-005e-448f-8d87-75db60ce5a5d.png">

tfcmt  -owner "owner" -repo repo-name -pr "93" plan -- terraform plan
linux ubuntu 22 amd

**Reply by suzuki-shunsuke** [reply]
Please check if GITHUB_TOKEN is set.

```sh
if [ -n "$GITHUB_TOKEN" ]; then
  echo "[DEBUG] GITHUB_TOKEN is set" >&2
else
  echo "[DEBUG] GITHUB_TOKEN isn't set" >&2
fi
tfcmt -owner "owner" -repo repo-name -pr "93" plan -- terraform plan
```

**Reply by vijyantg** [reply]
<img width="909" alt="image" src="https://user-images.githubusercontent.com/81227957/233400844-b49f65ca-3086-40bd-a3ff-ce2d9dec0f8c.png">

it is showing that the GH_T is set but still I m getting 
<img width="711" alt="image" src="https://user-images.githubusercontent.com/81227957/233400629-f6188c9b-17b1-4e96-aac5-25c733d45924.png">

**Reply by suzuki-shunsuke** [reply]
Hmm. It looks weird. Can you reproduce the issue in your laptop?
